include/boost/numeric/ublas/tensor/tensor_core.hpp

```diff
@@ -0,0 +1,786 @@
+//
+//  Copyright (c) 2018-2020, Cem Bassoy, cem.bassoy@gmail.com
+//  Copyright (c) 2019-2020, Amit Singh, amitsingh19975@gmail.com
+//
+//  Distributed under the Boost Software License, Version 1.0. (See
+//  accompanying file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+//  The authors gratefully acknowledge the support of
+//  Google and Fraunhofer IOSB, Ettlingen, Germany
+//
+
+
+/// \file tensor_core.hpp Definition for the tensor template class
+
+#ifndef BOOST_UBLAS_tensor_core_IMPL_HPP
+#define BOOST_UBLAS_tensor_core_IMPL_HPP
+
+#include <initializer_list>
+
+#include <boost/numeric/ublas/tensor/algorithms.hpp>
+#include <boost/numeric/ublas/tensor/expression.hpp>
+#include <boost/numeric/ublas/tensor/expression_evaluation.hpp>
+#include <boost/numeric/ublas/tensor/fixed_rank_extents.hpp>
+#include <boost/numeric/ublas/tensor/static_extents.hpp>
+#include <boost/numeric/ublas/tensor/dynamic_extents.hpp>
+#include <boost/numeric/ublas/tensor/strides.hpp>
+#include <boost/numeric/ublas/tensor/index.hpp>
```

Comment:
Should we internally use `"something"` instead of `<something>` for headers? 

The benefit would be that we can easily determine if a file is ublas header or one of ublas dependencies or standard.

Also, since "" includes are searched from current working directory first. Most of the IDE can find headers easily and we can type less long headers.

Reply:
This allows us to use any component of the tensor lib anywhere or makes it reusable.